packages/core/src/sharedWishlists/redux/middlewares/updateSharedWishlistUponItemsChanges.js

```diff
@@ -0,0 +1,41 @@
+import {
+  ADD_ITEM_TO_WISHLIST_SUCCESS,
+  DELETE_WISHLIST_ITEM_SUCCESS,
+  UPDATE_WISHLIST_ITEM_SUCCESS,
+} from '../../../wishlists/redux/actionTypes';
+import { doUpdateSharedWishlist } from '../actions';
+import { getSharedWishlistId } from '../selectors';
+import { getUser, getUserIsGuest } from '../../../entities/redux/selectors';
+import { putSharedWishlist as putSharedWishlistClient } from '../../client';
+/**
+ * Middleware to update the shared wishlist if a wishlist item is added, edited or deleted.
+ *
+ * @function updateSharedWishlistUponItemsChanges
+ * @memberof module:sharedWishlists/middlewares
+ *
+ * @param {object} store - Redux store at the moment.
+ *
+ * @returns {Function} Redux middleware.
+ */
+export default store => next => action => {
+  if (
+    action.type ===
+    (ADD_ITEM_TO_WISHLIST_SUCCESS ||
```

Comment:
This will not work, you will need to specify all the conditions like this:

```suggestion
    if ( action.type === ADD_ITEM_TO_WISHLIST_SUCCESS || action.type === UPDATE_WISHLIST_ITEM_SUCCESS...)
```

Reply:
Thanks, updated 👍